src/docs-components/ToggleDocs.tsx

```diff
@@ -0,0 +1,118 @@
+          { label: true, helperText: true },
+        ].map(({ label, helperText }) => (
+          <tet.div
+            px="$dimension-1000"
+            pb="$dimension-500"
+            key={`${label}${helperText}`}
+          >
+            <SectionHeader
+              variant="H2"
+              as="h3"
+              py="$dimension-500"
+              labels={getLabels(label, helperText)}
+            />
+            <States
+              states={[':normal', ':disabled']}
+              flexBasis="130px"
+              gap="$dimension-500"
+            />
+            <tet.div
+              display="flex"
+              alignItems="center"
+              justifyContent="center"
+              gap="$dimension-500"
+            >
+              <tet.div flexBasis="130px" flexGrow="1" flexShrink="0">
+                <tet.div
+                  display="flex"
+                  flexDirection="column"
+                  gap="$space-component-gap-2xLarge"
+                >
+                  <Toggle
+                    isIndeterminate={state === 'Indeterminate'}
+                    isChecked={state === 'Checked'}
+                    mt="$dimension-500"
+                    label={label ? 'Label' : (undefined as unknown as string)}
+                    helperText={
+                      helperText ? { text: 'Helper Text' } : undefined
+                    }
+                    size="large"
+                  />
+                  <Toggle
+                    isIndeterminate={state === 'Indeterminate'}
+                    isChecked={state === 'Checked'}
+                    mt="$dimension-500"
+                    label={label ? 'Label' : (undefined as unknown as string)}
+                    helperText={
+                      helperText ? { text: 'Helper Text' } : undefined
+                    }
+                  />
+                </tet.div>
+              </tet.div>
+              <tet.div flexBasis="130px" flexGrow="1" flexShrink="0">
+                <tet.div
+                  display="flex"
+                  flexDirection="column"
+                  gap="$space-component-gap-2xLarge"
+                >
+                  <Toggle
+                    isIndeterminate={state === 'Indeterminate'}
+                    isChecked={state === 'Checked'}
+                    mt="$dimension-500"
+                    state="disabled"
+                    label={label ? 'Label' : (undefined as unknown as string)}
+                    helperText={
+                      helperText ? { text: 'Helper Text' } : undefined
+                    }
+                    size="large"
+                  />
+                  <Toggle
+                    isIndeterminate={state === 'Indeterminate'}
+                    isChecked={state === 'Checked'}
+                    state="disabled"
+                    mt="$dimension-500"
+                    label={label ? 'Label' : (undefined as unknown as string)}
```

Comment:
This casting shouldn't be here. If something is wrong with the label in Toggle, we should fix it